Documentation/WritingReST/Index.rst

```diff
@@ -45,13 +45,12 @@ http://openalea.gforge.inria.fr/doc/openalea/doc/_build/html/source/sphinx/rest_
       <https://docs.typo3.org/typo3cms/drafts/github/TYPO3-Documentation/t3SphinxThemeRtdDemoDocs/Index.html>`__
 
 
-
-
 .. toctree::
    :hidden:
 
    Introduction
    BasicRestSyntax
+   ContentElements/Index
```

Comment:
Adding the new content elements as subpages of the new folder "ContentElements" is breaking the current organizing, where each content element is added as subpage to "Reference" and as content element to page "CheatSheet". Will continue with reviewing the single pages as soon as the reorganizing is done.